test/common/compressor/zlib_compressor_impl_test.cc

```diff
@@ -0,0 +1,89 @@
+#include "common/buffer/buffer_impl.h"
+#include "common/common/assert.h"
+#include "common/compressor/zlib_compressor_impl.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Compressor {
+namespace {
+
+class ZlibCompressorImplTest : public testing::Test {
+protected:
+  static const std::string get200CharsText() {
+    std::string str{"Lorem ipsum dolor sit amet, consectetuer "};
+    str += "adipiscing elit. Aenean commodo montes ridiculus ";
+    str += "ligula eget dolor. Aenean massa. Cum sociis Donec ";
+    str += "natoque penatibus et magnis dis parturient, nascetur ";
+    str += "mus qu.";
+    ASSERT(str.length() == 200);
```

Comment:
You can do:
```
     const std::string str{"Lorem ipsum dolor sit amet, consectetuer "
     "adipiscing elit. Aenean commodo montes ridiculus "
     "ligula eget dolor. Aenean massa. Cum sociis Donec "
     "natoque penatibus et magnis dis parturient, nascetur "
     "mus qu."};
```
To avoid concatenating std::string objects (string literals are concatenated if adjacent).